src/main/java/graphql/execution/ExecutionTypeInfo.java

```diff
@@ -34,15 +37,17 @@
     private final GraphQLFieldDefinition fieldDefinition;
     private final ExecutionPath path;
     private final boolean typeIsNonNull;
+    private final Map<String, Object> arguments;
     private final ExecutionTypeInfo parentType;
 
-    private ExecutionTypeInfo(GraphQLType type, GraphQLFieldDefinition fieldDefinition, Field field, ExecutionPath path, ExecutionTypeInfo parentType, boolean nonNull) {
+    private ExecutionTypeInfo(GraphQLType type, GraphQLFieldDefinition fieldDefinition, Field field, ExecutionPath path, ExecutionTypeInfo parentType, boolean nonNull, Map<String, Object> arguments) {
```

Comment:
as you said: It should named Differently: so please rename it :-) (or maybe in an extra PR if you prefer)